### PR TITLE
Merchant controller test

### PR DESCRIPTION
**Pull request by KKennedyCodes**

### CODE REVIEW

File: test/controllers/merchants_controller_test.rb [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
-  #   value(1+1).must_equal 2
-  # end
+  describe "index" do
+    it "should display all merchants" do
+      get merchants_path 
+      must_respond_with :success
+    end
+    
+    it "should not break if there are no merchants" do 
+      Merchant.all.destroy_all
```
Comment (dtingg) [inline test/controllers/merchants_controller_test.rb]:
```suggestion
      Merchant.destroy_all
```

File: test/fixtures/merchants.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+merchant_one:
+  uid: 123
+  username: george
+  email: anemail@adadev.org
+merchant_two:
+  uid: 1234
+  username: murphy
+  email: murc@adadev.org
+merchant_thre:
```
Comment (raisahv) [inline test/fixtures/merchants.yml]:
```suggestion
merchant_three:
```

**Review by raisahv — CHANGES_REQUESTED** [review]
One edit to fixture

**Comment by raisahv** [comment]
One change requested, other than that looks great

**Pull request merged**